beacon-light-client/plonky2/input_fetchers/balance_verification/withdrawal_credentials/runnable/light_cleaner.ts

```diff
@@ -25,7 +25,7 @@ import { CommandLineOptionsBuilder } from '../../../utils/cmdline';
   for (let i = 0; i < 39; i++) {
     queues.push(
       new WorkQueue(
-        new KeyPrefix(`${protocol}:${CONSTANTS.balanceVerificationQueue}:${i}`),
+        new KeyPrefix(`${protocol}:${CONSTANTS.depositBalanceVerificationQueue}:${i}`),
```

Comment:
Why are these changed? Maybe add the queue constant as a parameter to the whole script?